src/Facebook/InstantArticles/Elements/InstantArticle.php

```diff
@@ -219,6 +219,64 @@ public function withFooter($footer)
     }
 
     /**
+     * Replace all the children within this InstantArticle
+     *
+     * @param Element[] $children Array of elements replacing the original.
+     *
+     * @return $this
+     */
+    public function withChildren($children)
+    {
+        Type::enforceArrayOf(
+            $children,
+            [
+                Ad::getClassName(),
+                Analytics::getClassName(),
+                AnimatedGIF::getClassName(),
+                Audio::getClassName(),
+                Blockquote::getClassName(),
+                Image::getClassName(),
+                H1::getClassName(),
+                H2::getClassName(),
+                Interactive::getClassName(),
+                ListElement::getClassName(),
+                Map::getClassName(),
+                Paragraph::getClassName(),
+                Pullquote::getClassName(),
+                RelatedArticles::getClassName(),
+                Slideshow::getClassName(),
+                SocialEmbed::getClassName(),
+                Video::getClassName()
+            ]
+        );
+        $this->children = $children;
+
+        return $this;
+    }
+
+    /**
+     * Replace all the children within this InstantArticle
+     *
+     * @param Type::INTEGER $index The index of the element to be deleted
+     *                             in the array of children.
+     *
+     * @return $this
+     */
+    public function deleteChild($index)
+    {
+        Type::enforce($index, Type::INTEGER);
+        $children = null;
```

Comment:
$children should never be null.

You gotta initialize it with:
`$children = [];`

See the class initialization. If the index informed is the only child of the array, this will assign a null array to the `$this->children` leading to a broken status of this element.